src/cljs/nr/pending_game.cljs

```diff
@@ -85,12 +88,16 @@
         (tr [:lobby.swap "Swap sides"])
         [:b.caret]]
        (into
-         [:ul.dropdown-menu.blue-shade]
-         (for [side ["Any Side" "Corp" "Runner"]]
-           [:a.block-link
-            {:on-click #(ws/ws-send! [:lobby/swap {:gameid @gameid
-                                                   :side side}])}
-            [:li (tr-side side)]]))])))
+        [:ul.dropdown-menu.blue-shade]
+        (for [side ["Any Side" "Corp" "Runner"]]
+          (if (current-side? @players side)
```

Comment:
@NoahTheDuke if you know of a more succint way to do this let me know, it's my first time touching Reagent!

Reply:
Two thoughts: 1) Lift the `(-> players first :side)` into a let block so all you're checking is `(if (= player-side side) ,,,)`, and 2) move the `if` to only wrap the properties map: `[:a.block-link (if (= player-side side) {:style ...} {:on-click ...}) [:li (tr-side side)]]`

Then we don't even need the `current-side?` function.